src/ApiPort/CommandLine/AnalyzeOptions.cs

```diff
@@ -117,6 +119,17 @@ private void UpdateRequestFlags(Options options)
                     RequestFlags |= AnalyzeRequestFlags.ShowBreakingChanges;
                 }
 
+                if (options.ShowRetargettingIssues)
+                {
+                    RequestFlags |= AnalyzeRequestFlags.ShowRetargettingIssues;
+
+                    //ShowRetargettingIssues doesn't mean much without ShowBreakingChanges turned on
+                    if (!RequestFlags.HasFlag(AnalyzeRequestFlags.ShowBreakingChanges))
```

Comment:
I thought about doing it this way, but thought the logic of the code would be more clear this way since I am already setting the ShowBreakingChanges in the if statement above.